cookiecutter-xblock/{{cookiecutter.repo_name}}/README.rst

```diff
@@ -137,7 +137,7 @@ See `config instructions <https://github.com/openedx/i18n-tools#transifex-comman
 transifex credentials.
 
 See `transifex documentation <https://docs.transifex.com/integrations/django>`_ for more details about integrating
-django with transiflex.
+django with transifex.
```

Comment:
Same here.

Reply:
I don't get what you mean by "same here".

Reply:
Please link to https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html instead.